splice_machine/src/main/java/com/splicemachine/derby/stream/output/direct/DirectPipelineWriter.java

```diff
@@ -46,10 +46,11 @@
     private TxnView txn;
     private final OperationContext opCtx;
     private final boolean skipIndex;
+    protected String tableVersion;
 
     private RecordingCallBuffer<KVPair> writeBuffer;
 
-    public DirectPipelineWriter(long destConglomerate, TxnView txn, byte[] token, OperationContext opCtx, boolean skipIndex){
+    public DirectPipelineWriter(long destConglomerate, TxnView txn, byte[] token, OperationContext opCtx, boolean skipIndex, String tableVersion){
         this.destConglomerate=destConglomerate;
```

Comment:
`this.tableVersion = tableVersion` is probably missing

Reply:
Added.  This is just there for consistency with other pipelinewriters, but is not actually used.